tests/files_test_mcnp/dummy_xsdir

```diff
@@ -7,4 +7,5 @@ atomic weight ratios
  05/19/2155
 directory
 1001.44c 1.111111 many_xs/1001.555nc 0 1 4 55555 0 0 5.5555E+05
-1001.66c 1.111111 such_data/1001.777nc 0 1 4 55555 0 0 5.5555E+05 ptable
+1001.66c 1.111111 such_data/1001.777nc 0 1 4 55555 0 0 5.5555E+05 +
```

Comment:
I wonder if it makes sense to keep this line and add another one so that both cases are tested?

Reply:
(p.s. that would mean changing the tests as well)

Reply:
Can do! I've made some changes here, is that along the lines of what you were thinking @gonuke?